apps/evm/src/pages/Account/NewPage/Transactions/TransactionsList/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { Spinner, cn } from '@venusprotocol/ui';
+import type { GetAccountHistoricalTransactionsOutput, TransactionWithAsset } from 'clients/api';
+import { Pagination } from 'components';
+import { format, isToday, isYesterday } from 'date-fns';
+import { useTranslation } from 'libs/translations';
+import { Placeholder } from 'pages/Account/Placeholder';
+import { useMemo } from 'react';
+import { TransactionRow } from '../TransactionRow';
+
+const INITIAL_PAGE_INDEX = 1;
+const ITEMS_PER_PAGE_COUNT = 20;
+
+export interface TransactionsListProps {
+  transactions: GetAccountHistoricalTransactionsOutput['transactions'];
+  transactionsCount: number;
+  onPageChange: (newPage: number) => void;
+  isLoading: boolean;
+  className?: string;
+}
+
+export const TransactionsList: React.FC<TransactionsListProps> = ({
+  transactions,
+  transactionsCount,
+  isLoading,
+  onPageChange,
+  className,
+}) => {
+  const { t } = useTranslation();
+
+  const transactionsGroupedByDate = useMemo(() => {
+    const groupedByDate = transactions.reduce<Record<string, TransactionWithAsset[]>>(
+      (acc, txData) => {
+        const isTodayGroup = isToday(txData.transaction.txTimestamp);
+        const isYesterdayGroup = isYesterday(txData.transaction.txTimestamp);
+
+        let dayGroup = format(txData.transaction.txTimestamp, 'MMM dd, yyyy');
+
+        if (isTodayGroup) {
+          dayGroup = t('account.transactions.today');
+        }
+
+        if (isYesterdayGroup) {
+          dayGroup = t('account.transactions.yesterday');
+        }
+
+        const dayTxs = acc[dayGroup] ? [...acc[dayGroup], txData] : [txData];
+
+        return {
+          ...acc,
+          [dayGroup]: dayTxs,
+        };
+      },
+      {},
+    );
+
+    return groupedByDate;
+  }, [transactions, t]);
+
+  if (isLoading) {
+    return <Spinner />;
+  }
+
+  if (transactions.length === 0) {
+    return (
+      <Placeholder
+        iconName="transactionFile"
+        title={t('account.transactions.placeholder.title')}
+        description={t('account.transactions.placeholder.description')}
+      />
+    );
+  }
+
+  return (
+    <>
+      <ul
+        className={cn(
+          'flex flex-col w-full items-center justify-evenly space-y-6 md:bg-cards md:p-4 md:rounded-xl',
+          className,
+        )}
+      >
+        {Object.keys(transactionsGroupedByDate).map(day => (
+          <div className="flex flex-col w-full space-y-3">
```

Comment:
According to the designs, there's no spacing between each list item on md and up. Instead, they each have a fixed height (64px) assigned to them (I'll leave another comment to update the `TransactionRow` component).

```suggestion
      <ul
        className={cn(
          'flex flex-col w-full items-center justify-evenly space-y-6 md:bg-cards md:py-4 md:rounded-xl md:space-y-0',
          className,
        )}
      >
        {Object.keys(transactionsGroupedByDate).map(day => (
          <div className="flex flex-col w-full space-y-3">
```